### PR TITLE
Task 1 - Write Script to create azure resources

### PR DESCRIPTION
**Pull request by AnthonySDi**
commented out code can be used to delete resources.

Hard coded file will be removed once this gets converted to an automation process with templates ARMs.

### CODE REVIEW

**Comment by elb-dev** [comment]
This looks fine to me, but it's not really easy to test without instructions, plus it's not on the team3 repo.

File: automation/azureResources.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+import os
+from haikunator import Haikunator
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.sql import SqlManagementClient
+
+REGION = 'westus2'
```
Comment (OyunominJuno) [inline automation/azureResources.py:7]: hard coding variable values are not recommended. you can look at my code for asking parameters from the user

**Review by OyunominJuno — COMMENTED** [review]
Agreed with Derek. Is there any instruction on how to run this script?

**Comment by sunshinejen** [comment]
Closing , new script has been done in Sprint 2

**Pull request closed without merging**